src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+export const GENDER_MALE = 'm';
+export const WIFE = 'wife';
+export const HUSBAND = 'husband';
+
+export const Person = ({ person:
+  { name, age, sex, isMarried, partnerName } }) => {
+  let getStatus;
```

Comment:
```suggestion
  const partner = sex === GENDER_MALE ? WIFE : HUSBAND;
```
It can be simplified